AddTestGuide.md

```diff
@@ -0,0 +1,197 @@
+# USF OpenTripPlanner Adding Testing cases instructions
+
+This readme is intend to provide information solely about adding a new test case. For more information about how to use this test unit please refer to the main readme (It can be found here: https://github.com/CUTR-at-USF/test/tree/WIP-jmfield2#usf-opentripplanner-testing-framework).
+
+
+## The test cases can be found in the following spreadsheet:
+
+https://docs.google.com/spreadsheets/d/1f_CTDgQfey5mY1eMO03D7UZ8855D-mxHsfYfsA3c4Zw
+
+## Adding a test case
+
+Note: To add a new case the right to edit the spreadsheet is necessary (Link above).
+
+**Example:**
+
+```
+$ python test_runner.py --add "http://mobullity.forest.usf.edu/index.html?module=planner&fromPlace=28.06914026483356%2C-82.40278244018555&toPlace=28.069272801561006%2C-82.40248203277588&time=6%3A09pm&date=07-23-2015&mode=TRANSIT%2CWALK&maxWalkDistance=1609.34&wheelchair=false&arriveBy=false&bannedRoutes=undefined&showIntermediateStops=true&itinIndex=0" --add-class=USFPlanner -R
+```
+
+The script ```test_runner.py``` need to be run with the following attribut:
+
+```
+--add ""
+```
+
+The itinerary link retrieved from the planner need to be inside of the quote. 
+To copy the itinerary link from the planner, first plan your trip then in the bottom of the detail of this trip there should be a link named "Link to Itinerary", you only need to copy the url of this link.
+
+```
+--add-class=USFPlanner -R
+```
+## Populating the various fields
+
+Note: When the spreadsheet gets large, it may be confusing because some rows will reference tests but the column will still exist for other rows ... the current way to deal with this is just to add a single "-" in the column to "suppress" that test.
+
+
+###The following field are automatically added by the --add command:
+
+**Description of the test**
+
+This field need to contain a brief description of the test.
+
+column: desciption <br/>
+value: a string of character
+
+Note: this field is automatically filled up but it needs to be modify to something specific.
+
+**Coordonate for the path**
+
+Start and end location of the trip to test.
+
+columns: fromPlace **and** toPlace
+value: two float separated by a coma (Latitude, Longitude)
+
+**Check mode of transit**
+
+<!---TODO-->
+
+column: mode <br/>
+value: name of the modes of transit comma-delimited 
+
+**Check max walk distance**
+
+<!---TODO-->
+
+column: maxWalkDistance <br/>
+value: float
+
+**Check arriveBY**
+
+<!---TODO-->
+
+column: arriveBy <br/>
+value: boolean
+
+**Check Intermeediate stops**
+
+<!---TODO-->
+
+column: showIntermediateStops <br/>
+value: boolean
+
+**Time**
+
+<!---TODO-->
```

Comment:
"The time that the trip should depart (or arrive, for requests where arriveBy is true)."

The date column OTP parameter may be more influential in the routing process than time -- since some buses don't run on the weekend, etc